src/test/java/seedu/address/model/util/attributes/AmountTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.util.attributes;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class AmountTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Amount(null));
+    }
+
+    @Test
+    public void constructor_invalidAmount_throwsIllegalArgumentException() {
+        String invalidAmount = "";
+        assertThrows(IllegalArgumentException.class, () -> new Amount(invalidAmount));
+    }
+
+    @Test
+    public void isValidAmount() {
```

Comment:
Should we follow the three part format for naming of test methods?

Reply:
Yup its quite referenced from the original AB3 tests